<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NplCadWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>Design</string>
    </property>
    <addaction name="actionBuild"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDesign"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="DockFiles">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>46</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="Dock3D">
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>46</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>3D View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="DockLog">
   <property name="minimumSize">
    <size>
     <width>102</width>
     <height>260</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../qnplcad.qrc">
     <normaloff>:/res/images/file-new.png</normaloff>:/res/images/file-new.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qnplcad.qrc">
     <normaloff>:/res/images/file-open.png</normaloff>:/res/images/file-open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="../qnplcad.qrc">
     <normaloff>:/res/images/transform-mode-local.png</normaloff>:/res/images/transform-mode-local.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qnplcad.qrc">
     <normaloff>:/res/images/file-save.png</normaloff>:/res/images/file-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="../qnplcad.qrc">
     <normaloff>:/res/images/file-save-all.png</normaloff>:/res/images/file-save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qnplcad.qrc"/>
 </resources>
 <connections/>
</ui>
